### ISSUE TITLE
uses() does not create short named class aliases

### ISSUE THREAD

**Issue by kiesel** [xp-framework/core#48]
The following snippet should actually work:

```sh
# kiesel at vb-kiesel in ~/dev/xp6/core on git:master o [17:31:33]
$ USE_XP=. xp -w 'uses("util.Date"); return new Date();'
PHP Fatal error:  Class 'Date' not found in /home/idev/dev/xp6/core/src/main/php/xp/runtime/Dump.class.php(45) : eval()'d code on line 1
PHP Stack trace:
PHP   1. {main}() /home/idev/bin/class-main.php:0
PHP   2. lang\reflect\Method->invoke() /home/idev/bin/class-main.php:259
PHP   3. ReflectionMethod->invokeArgs() /home/idev/dev/xp6/core/src/main/php/lang/reflect/Method.class.php:93
PHP   4. xp\runtime\Dump::main() /home/idev/dev/xp6/core/src/main/php/lang/reflect/Method.class.php:93
PHP   5. eval() /home/idev/dev/xp6/core/src/main/php/xp/runtime/Dump.class.php:45
Exception lang.Error (Fatal error)
  at <main>::xp\{closure}() [line 1 of Dump.class.php(45) : eval()'d code] Class 'Date' not found
  at <main>::xp\{closure}() [line 0 of StackTraceElement.class.php]

```

**Comment by kiesel** [comment]
First attempt in fixing:

```diff
diff --git a/src/main/php/lang.base.php b/src/main/php/lang.base.php
index 4cbb907..eb6f244 100755
--- a/src/main/php/lang.base.php
+++ b/src/main/php/lang.base.php
@@ -390,6 +390,16 @@ function uses() {
       }
       call_user_func([$class, '__import'], $scope);
     }
+
+    $short= substr($str, strrpos($str, '.')+ 1);
+    if (!class_exists($short, false)) {
+      class_alias(strtr($str, '.', '\\'), $short);
+      \xp::$cn[$short]= $str;
+    }
   }
 }
 // }}}
```

**Comment by thekid** [comment]
This should have surfaced a long time ago, but better late than never :grinning:

**Comment by thekid** [comment]
Please try applying  #49 to see if it solves your problem; we might need to add more test cases e.g. for nested loading.

**Comment by kiesel** [comment]
Applied it and tested my usecase; this seems to work AFAICT.

**Comment by thekid** [comment]
Merged #49
